docs/components/modeler/forms/form-element-library/forms-element-library-button.md

```diff
@@ -0,0 +1,16 @@
+---
+id: forms-element-library-button
+title: Button
+description: A form element to trigger form actions
+---
+
+A button allowing the user to trigger form actions.
+
+![Form Button Symbol](/img/form-icons/form-button.svg)
+
+### Configurable properties
+
+- **Field Label**: Label to be displayed on top of the button.
```

Comment:
Please lowercase "Label" here, since in the new properties panel, it should also be sentence case, not title case. Same for all other properties panel references